Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -4121,6 +4121,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         new DiscretePropertyWrapper<const Vector<Style::ScopedName>&>(CSSPropertyAnchorName, &RenderStyle::anchorNames, &RenderStyle::setAnchorNames),
         new DiscretePropertyWrapper<const std::optional<Style::ScopedName>&>(CSSPropertyPositionAnchor, &RenderStyle::positionAnchor, &RenderStyle::setPositionAnchor),
         new DiscretePropertyWrapper<Style::PositionTryOrder>(CSSPropertyPositionTryOrder, &RenderStyle::positionTryOrder, &RenderStyle::setPositionTryOrder),
+        new DiscretePropertyWrapper<const Vector<PositionTryFallback>&>(CSSPropertyPositionTryFallbacks, &RenderStyle::positionTryFallbacks, &RenderStyle::setPositionTryFallbacks),
```

Comment:
The corresponding change in `RenderStyle::conservativelyCollectChangedAnimatableProperties` is missing. `transition: all` won't work correctly without it: https://searchfox.org/wubkat/rev/d46c14cf9008c4385cf0ac7919561b278d047e85/Source/WebCore/rendering/style/RenderStyle.cpp#1482

Reply:
Probably worth an interpolation test, which would have caught this.

Reply:
separate issue

Reply:
Doesn't seem separate to me, seems broken to have `transition: position-try-fallback` work but not 1transition: all broken1

Reply:
It's ok landing this because this is still disabled, but generally doesn't make sense to separate two things that are tied.

Reply:
It really doesn't make sense to try to implement everything in a single patch while building up things. The animation part is here only because tests crash otherwise.

Reply:
Good point about lack of transition: all test for this though, mind filing a bug?